test/fixture/locators.test.ts

```diff
@@ -0,0 +1,138 @@
+import * as playwright from '@playwright/test'
+import * as path from 'path'
+import {
+  locatorFixtures as fixtures,
+  LocatorFixtures as TestingLibraryFixtures,
+} from '../../lib/fixture'
+
+const test = playwright.test.extend<TestingLibraryFixtures>(fixtures)
+
+const {expect} = test
+
+test.describe('lib/fixture.ts (locators)', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(`file://${path.join(__dirname, '../fixtures/page.html')}`)
+  })
+
+  test('should handle the query* methods', async ({queries: {queryByText}}) => {
+    const element = queryByText('Hello h1')
```

Comment:
IMHO, rename the returns in this file `const locator` to make it clearer that they're `Locator`s.